### DISCUSSION TITLE
Welcome to CityEnergyAnalyst Discussions!

### DISCUSSION THREAD

**Discussion by shanshanhsieh** [architecture-building-systems/CityEnergyAnalyst#3144] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by preetambhu** [comment]
Hi there,
I am using Mumbai database and getting problem with supply data sheet.
Error is coming when I start running archetype mapper with supply sheet.
How can i over come this problem?
Thank you.
